virtManager/details/console.py

```diff
@@ -911,7 +917,7 @@ def _console_list_menu_toggled(self, src, dev):
 
     def _populate_console_list_menu(self, ignore=None):
         self._consolemenu.rebuild_menu(
-                self.vm, self._console_list_menu,
+                self.vm, self.config, self._console_list_menu,
```

Comment:
All this passing `self.config` around shouldn't be required when you move the functions out of config.py. Maybe they would need to be made global or something

Reply:
I have fixed it.